t/ui/03-setting-links.t

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env perl
```

Comment:
Maybe you can add this file and the other test file to one of the existing tests? I know that it makes generally sense to avoid a file from becoming too big. However, the initialization of the test database and Selenium are unfortunately quite an overhead and splitting the tests too much makes it worse. It would maybe make sense to combine at least the two tests you're adding into one file.

Note that this test is not using Selenium at all so we would usually prefer to add it under just `t/` (and *not* `t/ui/`). (I know, that rule is already broken I still like to keep the structure.)

Reply:
merged it with the 28-keys_to_render_as_links.t and removed it